crates/iceberg/src/spec/snapshot.rs

```diff
@@ -84,7 +84,7 @@ pub struct Snapshot {
 /// Type to distinguish between a path to a manifestlist file or a vector of manifestfile locations
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone)]
 #[serde(untagged)]
-pub enum ManifestList {
+pub enum ManifestListLocation {
```

Comment:
It's conflict with `struct ManifestList` so I change it. I'm glad to use another better name or way to fix it.